git-project

```diff
@@ -32,6 +32,44 @@ if not os.path.isfile('.gitproj'):
     print '.gitproj file missing. Refer to the git-project readme'
     finish(1)
 
+# use the cwd .gitproj if exists
+if os.path.exists(cwd+'/.gitproj'):
+    # if there is a gitproj at cwd use it
+    root = cwd
+    os.chdir(root)
+else:
+    # look for .gitproj in the parent dir, ask for comfirmation if found
+    cwd_tmp = cwd
+    last_slash = cwd_tmp.rfind('/')
+    while (last_slash != -1):
+        cwd_tmp = cwd_tmp[:last_slash]
+        gitproj_tmpdir = cwd_tmp + '/.gitproj'
+        if cwd_tmp.lower() == root.lower():
+            print('Hit the root of the project. Uses project root .gitproj file. Halt.')
+            break
+        if os.path.isfile(gitproj_tmpdir):
+            load_comfirm = raw_input('Found gitproj at ' + gitproj_tmpdir +
+                                      '\nAre you sure you want to use it(y/n)?')
+            found_gitproj = False
+            # make sure user gives either y/n
+            while ((load_comfirm != 'y' or load_comfirm != 'n') and not found_gitproj):
+                if load_comfirm.lower() == 'y':
+                    root = cwd_tmp
+                    os.chdir(root)
+                    found_gitproj = True
+                    break
+                elif load_comfirm.lower() == 'n':
+                    print 'continue searching'
+                    break
+                else:
+                    load_comfirm = raw_input(' invalid input ' + load_comfirm +
```

Comment:
Can we wrap the input prompts under a flag? Something like:

`git-project --prompt`

Only because git-project needs to be able to run autonomously when deploying/testing etc.